model.py

```diff
@@ -862,10 +861,15 @@ def plot_gmm_prior(stellar_type_prior):
         quantiles=[0.16, 0.5, 0.84],
         levels=[0.68,0.95,0.99],
         show_titles=True,
-        pcolor_kwargs = {'cmap':cm.Blues}
+        color='midnightblue'
+        #pcolor_kwargs={'cmap':cm.Blues}
```

Comment:
This line threw an error for me, and I don't actually see `pcolor_kwargs` listed in the documentation for `corner.corner`. The color `midnightblue` is similar to the color used in the colormap `Blues`.